### PR TITLE
Bug 1351082 - Disable prefetch

### PR DESCRIPTION
**Pull request by nwgh**
This is a system add-on to disable predictive prefetch. See https://bugzilla.mozilla.org/show_bug.cgi?id=1351082 (and associated bugs) for more details.

### CODE REVIEW

**Comment by Standard8** [comment]
@nwgh Please can you rebase this, so that it picks up the latest master and tests can run properly.

File: addons/disable-prefetch/install.rdf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+        <em:minVersion>52.0</em:minVersion>
+        <em:maxVersion>52.*</em:maxVersion>
+      </Description>
+    </em:targetApplication>
+
+    <!-- Front End MetaData -->
+    <em:name>Disable Prefetch</em:name>
+    <em:description>Disables predictive prefetch by default in 52.*.</em:description>
+  </Description>
```
Comment (ckprice) [inline addons/disable-prefetch/install.rdf]: Missing `</RDF>`, not sure if this would block releasing, @felipc?

Comment (felipc) [inline addons/disable-prefetch/install.rdf]: Ah sorry.. yeah this error is ignored but it throws a warning in the console, so let's fix it

**Comment by Standard8** [comment]
Rebasing on latest has picked up the very latest change to allow taskcluster to run, since @felipc has already approved this, I'll merge it now.

**Pull request merged**